systemservices/marketplace/mesg.yml

```diff
@@ -49,18 +49,26 @@ tasks:
                     description: ""
                     type: String
                   manifest: &manifest
-                    name: ""
-                    description: ""
-                    type: String
-                  manifestProtocol: &manifestProtocol
-                    name: ""
-                    description: ""
-                    type: String
-                  manifestData: &manifestData
                     name: ""
                     description: ""
                     type: Object
                     optional: true
+                    object: 
+                      version: { type: String }
+                      service: &serviceInformation
+                        type: Object
+                        object:
+                          hash: { type: String }
+                          hashVersion: { type: String }
+                          deployment:
+                            type: Object
+                            object:
+                              type: { type: String }
+                              source: { type: String }
+                          definition: { type: Object }
```

Comment:
why using `{ }` style?

Reply:
just more compact no other reasons

Reply:
Can you keep the same style as the rest of the file